src/kaboom.ts

```diff
@@ -477,7 +476,9 @@ export default (gopt: KaboomOpt = {}): KaboomCtx => {
 			const tex = new Texture(0, 0, opt)
 			tex.bind()
 			gl.texImage2D(gl.TEXTURE_2D, 0, gl.RGBA, gl.RGBA, gl.UNSIGNED_BYTE, img)
+			// @ts-ignore
 			tex.width = img.width
+			// @ts-ignore
```

Comment:
why do we have to ignore here?

Reply:
tried remove them and don't get any warnings

Reply:
I see this one ![image](https://github.com/replit/kaboom/assets/71136486/7f363c9e-8619-4933-a8d4-606d4e99026e)